codecov.yml

```diff
@@ -13,4 +13,4 @@ coverage:
 
 codecov:
     notify:
-        after_n_builds: 7
+        after_n_builds: 8
```

Comment:
What is this magical number?

Reply:
The number of build submitted to codecov.io before it sends a notification here on the GitHub PR thread